src/components/elements/carousel/carousel-style.js

```diff
@@ -78,3 +90,62 @@ export const StyledNextButton = styled(StyledPrevButton)`
         right: -12px;
     }
 `
+
+export const CardContent = styled(Text)`
+    display: none;
+    font-size: 14px;
+    text-align: center;
+
+    @media ${device.tablet} {
+        display: inherit;
+        padding: 8px 11px;
+    }
+
+    @media ${device.laptop} {
+        padding: 8px 32px;
+    }
+`
+
+export const BottomCard = styled(VFlex)`
+    width: 100px;
+    height: 74px;
+    align-items: center;
+    justify-content: center;
+    background-color: ${({ theme }) => theme.colors.background_default};
+    border-left: 1px solid rgba(229, 229, 229, 0.8);
+    border-bottom: ${props => (props.selected ? 'unset' : '1px solid rgba(229, 229, 229, 0.8)')};
+    border-top: ${props => (props.selected ? '2px solid' : '1px solid')};
+    border-top-color: ${props =>
+        props.selected ? ({ theme }) => theme.colors.primary : 'rgba(229, 229, 229, 0.8)'};
+    box-shadow: ${props => (props.selected ? '0 32px 26px -24px rgba(0,0,0,0.1)' : 'unset')};
+    transition: border-top-color 0.5s, background-color 0.5s, border-bottom 0.5s, box-shadow 0.5s;
+
+    :nth-last-child(1) {
+        border-right: 1px solid rgba(229, 229, 229, 0.8);
+        box-shadow: ${props => (props.selected ? '18px 18px 28px -8px rgba(0,0,0,0.1)' : 'unset')};
```

Comment:
rgba(0,0,0,0.1) can be a boxshadow color?

Reply:
there seems to be no box-shadow-color property available and I can't seem to pass this as a variable in a prop at the box-shadow property so i don't think we need to add this to theme ?